pkg/image/resolve.go

```diff
@@ -54,3 +94,98 @@ func findStrings(obj map[string]interface{}, found map[string]bool) {
 		}
 	}
 }
+
+func getImagesFromCharts(path string) ([]string, error) {
```

Comment:
We could add another method or enhance this one to support get windows images from charts.